### DISCUSSION TITLE
How to exit a flet up on button click?

### DISCUSSION THREAD

**Discussion by halloleo** [flet-dev/flet#1732] [General]
I want to exit a Flet app when clicking an `ElevatedButton`. I tried 

```
def exit_click(e):
        sys.exit()

ft.ElevatedButton("Exit", on_click=exit_click)
```

but  `sys.exit()` does seem to do anything here.

Any sugegstion how to exit the app correctly?

**Comment by ndonkoHenri** [comment]
You can use these for that instead: [`page.window_close()`](https://flet.dev/docs/controls/page#window_close) or [`page.window_destroy)`](https://flet.dev/docs/controls/page#window_destroy)
